src/test/java/com/pubnub/contract/subscribe/steps/ThenSteps.kt

```diff
@@ -0,0 +1,51 @@
+package com.pubnub.contract.subscribe.steps
+
+import com.pubnub.api.models.consumer.pubsub.PNMessageResult
+import com.pubnub.api.models.consumer.pubsub.PNSignalResult
+import com.pubnub.contract.subscribe.state.SubscribeState
+import io.cucumber.java.en.Then
+import org.awaitility.Awaitility.await
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.*
+import org.junit.jupiter.api.Assertions.assertTrue
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+class ThenSteps(
+    private val subscribeState: SubscribeState
+) {
+
+    @Then("I receive 2 messages in my subscribe response")
+    fun I_receive_2_messages_in_my_subscribe_response() {
+        await()
+            .atMost(500, TimeUnit.MILLISECONDS)
+            .until {
+                subscribeState.messages.size == 2
+            }
+    }
+
+    @Then("response contains messages with {string} and {string} message types")
+    fun response_contains_messages_with_message_types(
+        messageTypeOfFirstMessage: String,
+        messageTypeOfSecondMessage: String
+    ) {
+        assertThat(subscribeState.messages.size, equalTo(2))
```

Comment:
This is already checked by the other step, and this one doesn't mention anything about how many should there be. But even if we ignore all those things it's checked by the next assertion. `containsInAnyOrder` will fail if there's more elements

Reply:
ok,I can delete this.